frontend/src/mocks/handlers/coupon.ts

```diff
@@ -1,27 +1,53 @@
 import { rest } from 'msw';
 
 import { BASE_URL } from '@/apis';
-import coupons from '@/mocks/fixtures/coupon';
-import users from '@/mocks/fixtures/users';
+import coupons, { FixtureCouponType } from '@/mocks/fixtures/coupon';
+import { COUPON_STATUS } from '@/types/coupon/client';
 import { ChangeCouponStatusRequest, CreateCouponRequest } from '@/types/coupon/remote';
 
 export const couponHandler = [
-  rest.get(`${BASE_URL}/coupons`, (req, res, ctx) => {
-    const { headers } = req;
+  rest.get(`${BASE_URL}/coupons/accept`, (req, res, ctx) => {
+    try {
+      const acceptedCoupons = coupons.findAcceptedCouponList();
+
+      const reservationObject = acceptedCoupons.reduce((prev, current) => {
+        if (!current.meetingDate) {
+          return prev;
+        }
+
+        const prevACceptedCoupons = prev[current.meetingDate] ?? [];
 
+        return { ...prev, [current.meetingDate]: [...prevACceptedCoupons, current] };
+      }, {} as Record<string, any[]>);
+
+      const reservationList = Object.entries(reservationObject).map(([key, value]) => ({
+        meetingDate: key,
+        coupons: value,
+      }));
+
+      return res(ctx.status(200), ctx.json({ data: reservationList }));
+    } catch ({ message }) {
+      return res(ctx.status(401), ctx.json({ error: message }));
```

Comment:
```suggestion
      return res(ctx.status(401), ctx.json({ message }));
```

서버 응답은 위와 같은 형태로 와서 수정이 필요할것같습니당!!~!

Reply:
해결!